extension/src/NABfunctions.ts

```diff
@@ -37,6 +38,7 @@ export async function refreshXlfFilesFromGXlf(
   suppressMessage = false
 ): Promise<void> {
   console.log("Running: RefreshXlfFilesFromGXlf");
+  Telemetry.trackEvent("refreshXlfFilesFromGXlf");
```

Comment:
can we make use  of `arguments.callee.name` here? It would guard against spelling errors and name changes :)

Reply:
I would have done that, if I knew about that functionality! 😁

I'll fix that.

Reply:
Skipping this, since `arguments.callee.name` is deprecated since ES5.